index.html

```diff
@@ -12,29 +12,66 @@
 	</head>
 	<body>
 		<h1>BATTLESHIP</h1>
-		<div class="board-container">
-			<input type="text" class="player-name" value="Player 1">
-			<table class="board" id="board0"></table>
-		</div>
-		<div class="board-container">
-			<input type="text" class="player-name" value="Player 2">
-			<table class="board" id="board1"></table>
-		</div>
-		<div id="message"></div>
-		<input type="button" id="switch-turn" value="Switch turn">
 
-		<div class="option">
+		<!-- Probably add CSS to this -->
+		<div style="text-align:center" id="menu">
+            <h2>WELCOME</h2>
+            <p>Please enter player's names: </p>
+            <form action="">
+                Player 1:  <input autofocus type="text" id="Player1"> <br>
+                Player 2:  <input type="text" id="Player2"> <br>
+			</form>
 			<p>Number of Ships: <span id="num-ships">1</span></p>
 			<input type="range" min="1" max="5" value="1" id="ship-slider">
 			<input type="button" id="complete" value="Done">
 		</div>
+		
+
+		<div class="both-boards" id="both_boards">
+			<div class="board-container">
+				<input type="text" class="player-name" id="firstPlayer">
+				<table class="board" id="board0"></table>
+			</div>
+			<div class="board-container">
+				<input type="text" class="player-name" id="secondPlayer">
+				<table class="board" id="board1"></table>
+			</div>
+			<div id="message"></div>
+			<input type="button" id="switch-turn" value="Switch turn">
+		</div>
+
+
+		<!-- Probably add CSS to this -->
+		<div id="modal" class="modal">
+			<div class="modal-content">
+				<p id=modal-content></p>
+			</div>
+		</div>
+
+
 
 		<script>
-			window.executive = new Executive(9, 9);
 
-			// For testing
-			executive.placeSampleShips();
-			executive.renderBoards();
+			 //wait for the DOM to load
+			document.addEventListener("DOMContentLoaded",()=>{
```

Comment:
We should avoid placing large amounts of code directly in the HTML file. I see two possible solutions: construct the Executive class without the number of ships and have it take care of these things, or have a new class that manages selecting the number of ships, completing the game, etc. In that case, the class that is currently called Executive could be renamed Game, as it would be responsible for everything that happens during gameplay, but not before and after. A new class is probably better for object-orientedness, but this is a rather significant change so it doesn't need to be done before I'd merge this pull request.

Reply:
Noted, I'll change it later then.